Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/RideRequestActivity.java

```diff
@@ -141,16 +210,20 @@ public void onViewHolderBind(SheetAdapter.SheetViewHolder holder, int position,
                 case "inc":
                     updateIndicators();
                     holder.baseFare.setText(String.valueOf(model.getBaseFare() + model.getUpdatedAmount()));
+                    holder.currency.setText(String.valueOf(model.getCurrency()));
+                    updateIncreaseDecreaseButtons(holder, model);
                     return;
                 case "time":
+                    updateAcceptButtonText(holder, model.getRideRequestPopupDelayDuration(), model.getStartTime(), (model.isGotoTag() ? getString(R.string.accept_goto) : getString(R.string.accept_offer)));
                     updateProgressBars(true);
                     return;
             }
 
             holder.pickUpDistance.setText(model.getPickUpDistance()+" km ");
             holder.baseFare.setText(String.valueOf(model.getBaseFare() + model.getUpdatedAmount()));
             holder.distanceToBeCovered.setText(model.getDistanceToBeCovered() + " km");
-            if( key.equals("yatrisathiprovider") ){
+
+            if( service.equals("yatrisathiprovider") ){
```

Comment:
let's figure out ways to have configs for native as well. Having hardcoded values is not scalable @ashishsingh101 @Rohit4997 @Vignesh-772

Reply:
sure